### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by yakov116**
- Remove duplicate page detect
- Sort types A-Z
- `add-tags-to-commits` => `tags-on-commits-list`
- Ensure descriptions end with a period
- Move `maxAge` to be on one line 
- Update `github-url-detection`

### CODE REVIEW

**Review by fregante — APPROVED** [review]

**Review by notlmn — APPROVED** [review]
Awesome, @yakov116! 👏

File: readme.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ Thanks for contributing! 🦋🙌
 - [](# "swap-branches-on-compare") [Adds link to swap branches in the branch compare view.](https://user-images.githubusercontent.com/857700/42854438-821096f2-8a01-11e8-8752-76f7563b5e18.png)
 - [](# "hide-zero-packages") [Hides the `Packages` tab in repositories if it’s empty.](https://user-images.githubusercontent.com/44045911/93543684-d1473b00-f98e-11ea-9957-e4464400f81b.png)
 - [](# "forked-to") [Adds a shortcut to your forks next to the `Fork` button on the current repo.](https://user-images.githubusercontent.com/55841/64077281-17bbf000-cccf-11e9-9123-092063f65357.png)
-- [](# "repo-age") [Adds the age of the repository to the statistics bar.](https://user-images.githubusercontent.com/3848317/69494069-7d2b1180-0eb7-11ea-9aa1-d4194e566340.png)
+- [](# "repo-age") [Adds the age of the repository to the stats/numbers bar.](https://user-images.githubusercontent.com/3848317/69494069-7d2b1180-0eb7-11ea-9aa1-d4194e566340.png)
```
Comment (fregante) [inline readme.md:127]: It’s not a bar anymore, perhaps:
```suggestion
- [](# "repo-age") [Displays the age of the repository in the sidebar.](https://user-images.githubusercontent.com/3848317/69494069-7d2b1180-0eb7-11ea-9aa1-d4194e566340.png)
```

Also perhaps we should avoid the generic “add” when possible, like in this case.

File: source/features/remove-upload-files-button.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ function init(): void {
 
 void features.add({
 	id: __filebasename,
-	description: 'Remove the "Upload files" button',
+	description: 'Remove the "Upload files" button.',
```
Comment (fregante) [inline source/features/remove-upload-files-button.tsx:14]: Always in third person, as if it said “[this feature] hides the button”
```suggestion
	description: 'Hides the "Upload files" button.',
```

However I’d drop this feature altogether. It’s just a minor cleanup feature that only affects GHE now. New PR for this

Comment (yakov116) [reply source/features/remove-upload-files-button.tsx:14]: I don't think I will have time to go though all the descriptions now, maybe in a week or 2. For now I made  sure they all have periods.

File: readme.md [hunk 3]
```diff
@@ -269,7 +269,7 @@ Thanks for contributing! 🦋🙌
 - [](# "remove-diff-signs") [Hides diff signs](https://user-images.githubusercontent.com/1402241/54807718-149cec80-4cb9-11e9-869c-e265863211e3.png) (since diffs are color coded already.)
 - [](# "suggest-commit-title-limit") [Suggest limiting commit titles to 72 characters.](https://user-images.githubusercontent.com/37769974/60379478-106b3280-9a51-11e9-88b9-0e3607f214cd.gif)
 - [](# "expand-all-collapsed-code") [Expands the entire file when you <kbd>alt</kbd> <kbd>click</kbd> on any "Expand code" button in diffs.](https://user-images.githubusercontent.com/44227187/64923605-d0138900-d7e3-11e9-9dc2-461aba81c1cb.gif)
-- [](# "add-tag-to-commits") [Displays the corresponding tags next to commits.](https://user-images.githubusercontent.com/14323370/66400400-64ba7280-e9af-11e9-8d6c-07b35afde91f.png)
+- [](# "tags-on-commits-list") [Displays the corresponding tags next to commits.](https://user-images.githubusercontent.com/14323370/66400400-64ba7280-e9af-11e9-8d6c-07b35afde91f.png)
```
Comment (yakov116) [inline readme.md:272]: Note to self: add to migrations

**Review by fregante — APPROVED** [review]
So far so good

**Comment by fregante** [comment]
uhm… a force push that undid a commit I just made

**Comment by yakov116** [comment]
> uhm… a force push that undid a commit I just made

Well that was making issues for me since I could not merge it since i merged master which already deleted that file!

**Comment by yakov116** [comment]
@fregante I think this round is done. 

Want to wait for something or add anything?

**Comment by fregante** [comment]
Looks good so far. Unless anything is waiting for this PR, it's always best to wait a few days just in case anything comes up. Lint is not urgent and doesn't affect the user anyway.

**Comment by yakov116** [comment]
> Looks good so far. Unless anything is waiting for this PR, it's always best to wait a few days just in case anything comes up. Lint is not urgent and doesn't affect the user anyway.

**Comment by fregante** [comment]
Then _all aboard the merge train!_

**Comment by yakov116** [comment]
Shipped

**Pull request merged**